### PR TITLE
Function to differenciate info, warnings and errors using colors in Generate, Import and Export

### PR DESCRIPTION
**Pull request by seralra96**
imagen

imagen

### CODE REVIEW

**Comment by m-kuhn** [comment]
I like this :+1: 
Can we leave info ones black (or a very dark gray)? They normally don't need extra attention.

File: projectgenerator/utils/qt_utils.py [hunk 1]
```diff
@@ -81,6 +81,21 @@ def selectFolder(line_edit_widget, title, parent):
 def make_folder_selector(widget, title=QCoreApplication.translate('projectgenerator', 'Open Folder'), parent=None):
     return partial(selectFolder, line_edit_widget=widget, title=title, parent=parent)
 
+def color_log_text(text, txt_edit):
+    textlines = text.splitlines()
+    for textline in textlines:
+        if textline.startswith("Info:"):
```
Comment (m-kuhn) [inline projectgenerator/utils/qt_utils.py]: Can we have this code somewhere in `libili2db`? we are actually parsing ili2db output here, it's not that much related to qt.

**Comment by m-kuhn** [comment]
@seralra96 the colors are also changed, looking at the codes, is this correct?

**Comment by seralra96** [comment]
Yep 😉, info and others are showing in dark gray,  Warnings in orange and Errors in red

**Comment by m-kuhn** [comment]
:tada:

**Pull request merged**